esp/esdllib/esdl_script.cpp

```diff
@@ -1047,7 +1048,6 @@ class CEsdlTransformOperationHttpPostXml : public CEsdlTransformOperationBase
                     break;
                 }
                 case XmlPullParser::END_TAG:
-                case XmlPullParser::END_DOCUMENT:
```

Comment:
@timothyklemm what happens if the xml is badly formed and the document ends without the closing tag being hit?

Reply:
Malformed XML results in parser exceptions.

Reply:
Just want to make sure we don't end up in an endless loop.  I think the reported issue was because END_DOCUMENT was checked in two places and would never get to the second.  If END_DOCUMENT can't happen without an exception that might be ok.  But if we can end up in a loop that's bad.

Reply:
In this and all changed cases, a start tag has been read and processing that element is in progress. To see an END_DOCUMENT before the balancing END_TAG implies malformed XML. The parser throws exceptions when malformed markup is parsed. No loops.